packages/manager/apps/vrack-services/src/pages/listing/components/VrackServicesDataGrid.tsx

```diff
@@ -29,27 +29,29 @@ import {
   RegionCell,
   ActionsCell,
 } from '@/components/VrackServicesDataGridCells';
-import { useUpdateVrackServices, useVrackServicesList } from '@/utils/vs-utils';
+import { useVrackServicesList } from '@/utils/vs-utils';
 import { urls } from '@/router/constants';
+import { useUpdateVrackServicesName } from '@/api/services/services-api-utils';
 
 export const VrackServicesDatagrid: React.FC = () => {
   const { t, i18n } = useTranslation('vrack-services/listing');
   const { trackClick, trackPage } = useOvhTracking();
   const navigate = useNavigate();
+
   const {
-    updateVS,
+    updateVSName,
     isPending,
+    error: updateNameError,
     isErrorVisible,
-    updateError,
     hideError,
-  } = useUpdateVrackServices({ key: 'listing' });
+  } = useUpdateVrackServicesName({});
```

Comment:
You should probably take the tracking from useUpdateVrackServices and add it in the `onSuccess` and `onError` callbacks